### ISSUE TITLE
Use scaled Beta for Uniform drift

### ISSUE THREAD

**Issue by null-a** [probmods/webppl#601]
As mentioned in [#308](https://github.com/probmods/webppl/issues/308#issuecomment-223065412).

**Comment by ngoodman** [comment]
now that we have easy-to-specify drift kernels, i wonder if we should remove the `uniformDrift` etc helpers?

**Comment by null-a** [comment]
My thinking with the helpers was that it's worth having them for common cases so we capture the knowledge about which distributions make good drift kernels for which distributions and how to parameterize the kernel based on previous value. Also, while having the flexibility to write your own is great, it's not trivial to do so. e.g. [`dirichletDrift`](https://github.com/probmods/webppl/blob/f1e39b0ade5b242b7ee4fb8cd59f02959d304fa9/src/header.wppl#L162)?

**Comment by ngoodman** [comment]
yeah, that's sensible. what if instead of providing a different Dist (`uniformDrift`) we make a default for the driftKernel (`sample(Uniform({..},{driftKernel: "default"})`)? maybe that's still too messy...?

**Comment by null-a** [comment]
> what if instead of providing a different Dist (uniformDrift) 

We don't provide a different distribution, `uniformDist` and friends are little webppl functions in the header?
